client/src/assets/scss/foundation/_settings.scss

```diff
@@ -74,6 +74,13 @@ $foundation-palette: (
   success: #3adb76,
```

Comment:
Perfect example of just a little bit of styling changes going a long way! 

**General**

- signout/signin buttons: text weight should be bolder
- 

**Homepage**
- main body elements should be more vertically aligned (lower)
- the swath of text here is too big. Users hate reading stuff unless its in article/blog form. Instead, you could break this into different subsections that appear in sections along the page (not unlike blue apron, or even the launchacademy.com website haha) 
- at the very least, trim down the width of the body text a little bit, and potentially even add an image to the background
- there are no calls to action (CTA) on this page (often best displated in the form of buttons, or anchor tags styled like buttons). How do I search for recipes? How do I view my pantry? You could even conditionally show these links/buttons to feature pages to unauthenticated or authenticated users e.g. a guest user doesnt have a link to a pantry. each of these CTAs could appear in different sections per my earlier comment even

Reply:
**Home/User Profile Page**
- What you call the "Home" page I likely wouldnt explicitly label as home, but instead "Profile" or "Your Profile"
- experiment with all-camps in buttons, and with slightly heavier font weight 
- I would rethink this page. Have two large boxes side by side. The left box has the text: "Your Recipes", overlaid on a nice image of a meal). Underneath the text you can smaller text of "Saved Recipes: 9"
- Right box should be the same width, and say "Ingredients in Your Pantry" or "Your Ingredients", overlaid on another nice image, but hopefully of various ingredients. It can have "Ingredients in Pantry: 24". 
- Think of how video game menu's are styled for inspiration

Reply:
**Recipe Show**

- love the use of two horizontally aligned boxes. This page needs IMAGES
- take inspiration from something like https://www.simplyrecipes.com/recipes/fattoush_bread_salad/
- the body content should be lower, slightly more vertically centered.
- each ingredient seems to have a description? I would just combine that in the same <p> or <span> tag as the ingredient quantity and title (see SimplyRecipes). The indentation looks odd otherwise
- I would remove the `:` colons after ingredients and steps
- experiment with rounded corners on the boxes, but decide for yourself!

Reply:
**User Recipe List Page**

- this page needs images too! I would rethink this from a list of links, to a list of squares  with images and titles, not unlike your search results page! If a recipe doesnt have a url (not from spoonacular) then you should add a default image e.g. http://thehalalworld.com/uploads/pages/Korean-Rice-Cake-and-Dumpling-Soup-Duk-Guk-and-Mandu.jpg

Reply:
**Recipe Search**
- this page rocks!
- I would remove the black border around each recipe box for sure. Instead, consider adding shadow to the image, or a very very light background to each image
- Titleize all of the recipe titles show on this page
- restyle the titles of the recipes to be bold and black. Then, add an on-hover affect to THE ENTIRE RECIPE BOX. To denote the user can click anywhere on the box to be linked to the recipe show page
- these recipe tiles are a little wide. Experiment with making them narrower, or even having a two column grid of recipes vertically

inspiration:https://www.simplyrecipes.com/recipes-5090746

Reply:
**Searched Recipe Show Page**

- the image is flush with the ingredients. add some distance between these elements
- hmmm that "Save this Recipe" button looks out of place. I would just add it below the image of the recipe, as well as at the bottom of the recipe
- Change "Save this Recipe" to "Save Recipe"